platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://nf-core-awsmegatests/proteinfold/results-9bea0dc4ebb26358142afbcab3d7efd962d3a820`
+    - A unique **Name** for the bucket, such as `nf-core-awsmegatests-proteinfold-h1065`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own pipeline data for interactive visualization, add the cloud bucket that contains the results of your nf-core/proteinfold pipeline run. See [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for more information. 
+:::
+
+#### Create a Jupyter notebook studio
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- In the **Compute & Data** tab:
+    - Select your AWS Batch compute environment. 
+    - Optional: Enter CPU and memory allocations. The default values are 2 CPUs and 8192 MB memory (RAM).
+        :::note
+        Studios compete for computing resources when sharing compute environments. Ensure your shared compute environment has sufficient resources to run both your pipelines and studio sessions. 
+        :::
+    - Mount data using Data Explorer: Mount the S3 bucket or directory path that contains the nf-core AWS megatests proteinfold data, or the pipeline work directory of your nf-core/proteinfold run. 
+- In the **General config** tab:
+    - Select the latest **Jupyter** container image template from the list.
+    - Optional: Enter a unique name and description for the data studio. 
+    - Check **Install Conda packages** and paste the following Conda environment YAML snippet:
+    ```yaml 
+    channels:
+    - schrodinger
+    - conda-forge
+    - bioconda
+    dependencies:
+    - python=3.10
+    - schrodinger::pymol-bundle
+    - conda-forge::libgl
+    - pip
+    - pip:
+      - biopython==1.85
+      - mdtraj==1.10.3
+    ```
+- Select **Add** or choose to **Add and start** the studio immediately.
+- If you chose to **Add** the studio in the preceding step, select **Connect** in the options menu to open the studio in a new browser tab. 
+
+#### Visualize protein structures 
+
+The following Python script visualizes and compares protein structures produced by Alphafold 2 and ESMFold, creating a composite interactive 3D image of the two structures with contrasting colors. The script retrieves the CA atoms from both structures, checks whether the structures have the same number of CA atoms, creates a contact map, calculates the per-residue RMSD for each structure, and visualizes the structures, secondary structures, and per-residue RMSD using PyMOL. 
+
+Run the following script in your Jupyter notebook to install the necessary packages and perform visualization:
+
+<details>
+<summary>Python script</summary>
+
+    1. Import libraries:
```

Comment:
Bulleting numbers are all `1.` 
I'm reading a raw file so I'm not sure if rendering engine will properly iterate step numbers or not. Calling out just in case.